shared/protocol/src/main/java/io/pravega/shared/segment/StreamSegmentNameUtils.java

```diff
@@ -366,4 +368,9 @@ public static boolean isTableSegment(String qualifiedName) {
         return tokens[1].equals(TABLES);
     }
     // endregion
+    
+    // region watermark
+    public static String getMarkForStream(String stream) {
```

Comment:
why do we need this additional wrapper method?

Reply:
I don't think we do. What are your thoughts on unifying those two files? (NameUtils and StreamSegmentNameUtils)

Reply:
We should definitely consolidate them. 
However, even if we do not unify them as part of this PR, StreamSegmentNameUtils is already visible publically. So we dont need to add this additional wrapper method here and directly access the method in StreamSegmentNameUtils.